Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apk add --no-cache bash
 ARG UNISON_VERSION=2.48.4
 RUN apk add --no-cache --virtual .build-dependencies build-base curl && \
     apk add --no-cache inotify-tools && \
+    apk add --no-cache shadow && rm /etc/default/useradd && \
```

Comment:
Why does `/etc/default/useradd` need to be removed?

Reply:
Good question.  Short answer is removing it eliminates warnings when running the useradd command.

Long answer - this file contains: ```
# useradd defaults file
GROUP=1000
HOME=/home
INACTIVE=-1
EXPIRE=
SHELL=/bin/bash
SKEL=/etc/skel
CREATE_MAIL_SPOOL=yes
```
On a barebones Alpine system such as this, there is no group with gid 1000. The home default is already set the `useradd` and so is unnecessary.  `useradd` also has reasonable defaults already set for INACTIVE and EXPIRE (account is active and never expires).  On this system we have no `/bin/bash`, nor `/etc/skel` folder.  And we don't need a mail spool created.  So in summary, this config is not appropriate for a bare bones distro such as Alpine.  Deleting it eliminates the following warnings:
```
bash-4.3# useradd gre
useradd: group '1000' does not exist
useradd: the GROUP= configuration in /etc/default/useradd will be ignored
Creating mailbox file: No such file or directory
```

And based on my testing so far, the container seems to work fine without it.